thirdparty/json/CMakeLists.txt

```diff
@@ -29,8 +29,7 @@ target_include_directories(nlohmann_json::nlohmann_json
 
 target_compile_options(nlohmann_json::nlohmann_json
     INTERFACE
-        $<$<CXX_COMPILER_ID:MSVC>:/experimental:external>
-        $<$<CXX_COMPILER_ID:MSVC>:/external:I$<BUILD_INTERFACE:${json_SOURCE_DIR}/include>>
+        $<BUILD_INTERFACE:$<$<CXX_COMPILER_ID:MSVC>:/external:I${json_SOURCE_DIR}/include>>
         $<$<CXX_COMPILER_ID:MSVC>:/external:W0>
         $<$<CXX_COMPILER_ID:MSVC>:/external:templates->
 )
```

Comment:
We're missing EOF here. I know that wasn't introduced by this change but we should fix it.

Reply:
No idea where it is lol - time to find it cuz the preview doesn't show it :^)

Reply:
![image](https://user-images.githubusercontent.com/25829582/200037793-357f90ef-da9c-49bd-8d7b-19d646ff3b6c.png)